agent/consul/health_endpoint.proto

```diff
@@ -3,13 +3,24 @@ syntax = "proto3";
 package consul;
 
 service Health {
```

Comment:
I think we should keep this generic - make the service just be "Consul" for now. As we add things I don't think there is value in splitting it into separate services since it won't be possible to have Consul split really.